src/widgets/queryinput.coffee

```diff
@@ -47,9 +47,9 @@ class QueryInput extends Widget
     @controller = @controller.concat controller
 
   setElement: (elements) ->
-    @element = ($ elements).map (elem) =>
+    @element = ($ elements).filter (elem) =>
```

Comment:
why this change? (just to know)

Reply:
Because map not work as expected, and tests are failing because this.